hnn_core/tests/test_extracellular.py

```diff
@@ -41,9 +41,11 @@ def test_extracellular_api():
     # all remaining input arguments checked by ExtracellularArray
 
     rec_arr = ExtracellularArray(electrode_pos)
-    with pytest.raises(AttributeError, match="can't set attribute"):
+    with pytest.raises(AttributeError,
+                       match="has no setter|can't set attribute"):
```

Comment:
Nice! Thanks for figuring out this OR condition. Could you add a comment on why we use an OR here. Maybe something about how >=3.11 changed the error msg to "has no setter".

Reply:
Done